src/components/TaskList.tsx

```diff
@@ -42,21 +87,18 @@ export default React.memo(function TaskList(props: PropsType) {
   const itemList = sortedEntries.map((entry) => {
     const uid = entry.uid;
 
-    return (
-      <TaskListItem
-        key={uid}
-        entry={entry}
-        onClick={props.onItemClick}
-      />
-    );
+    return <TaskListItem key={uid} entry={entry} onClick={props.onItemClick} onSave={props.onItemSave} />;
```

Comment:
Actually, another small comment that is better said now. This is one example (but I saw it a few times in this patch). Please don't change the formatting when you don't need to and just conform to what's already there. For example, here you changed the formatting which makes the diff unnecessarily long and makes it harder than it needs to be to see that you just added a property (on save) and passed it.

Reply:
ESLint is not complaining here, so I don't think it's eslint, I think you have some additional rules in your VSCode or whatever IDE you are using...